x-pack/filebeat/input/azureeventhub/_meta/fields.yml

```diff
@@ -0,0 +1,26 @@
+- key: azure-eventhub
+  title: "azure-eventhub"
+  description: >
+    Fields regarding the Azure event hub information.
+  fields:
+    - name: azure-eventhub
```

Comment:
We should [use underscore to combine words](https://www.elastic.co/guide/en/beats/devguide/7.9/event-conventions.html), but in any case I think these metrics should be stored under `azure.eventhub` instead of `azure-eventhub`. But changing this would be a breaking change (unless we duplicate fields), we would have to decide when to do it. Most likely in 8.0.